src/applications/disability-benefits/686c-674/config/form.js

```diff
@@ -1,15 +1,16 @@
 import fullSchema from 'vets-json-schema/dist/686C-674-schema.json';
+
 import environment from 'platform/utilities/environment';
 import preSubmitInfo from 'platform/forms/preSubmitInfo';
 import { VA_FORM_IDS } from 'platform/forms/constants';
 import { TASK_KEYS, MARRIAGE_TYPES } from './constants';
 import { isChapterFieldRequired } from './helpers';
-import { customTransformForSubmit } from './utilities';
```

Comment:
Do we still need this customTransformForSubmit?

Reply:
Nope, I removed this